src/probability_of_match_assuming_statistical_dependence.py

```diff
@@ -1,109 +1,96 @@
-from .download_swaps import get_swaps
+"""
```

Comment:
Thanks @marco for this great documentation! and refactoring of existing functions